Plugin.fs

```diff
@@ -9,39 +9,51 @@ open Dalamud.Game.ClientState
 open Dalamud.IoC
 open DalamudPluginProjectTemplateFSharp.Attributes
 
-type Plugin([<RequiredVersion("1.0")>] pluginInterface : DalamudPluginInterface,
-            [<RequiredVersion("1.0")>] commands : CommandManager,
-            [<RequiredVersion("1.0")>] chat : ChatGui,
-            [<RequiredVersion("1.0")>] clientState : ClientState) as this =
-    let mutable commandManager : PluginCommandManager<Plugin> = null
-    let mutable config = Configuration()
-    let ui = PluginUI()
+type Plugin
+    (
+        [<RequiredVersion("1.0")>] pluginInterface: DalamudPluginInterface,
+        [<RequiredVersion("1.0")>] commands: CommandManager,
+        [<RequiredVersion("1.0")>] chat: ChatGui,
+        [<RequiredVersion("1.0")>] clientState: ClientState
```

Comment:
There's no nice way to use the property initializer format from F# as that'd necessarily require `mutable`s that can be avoided in the constructor format.